fastods/src/main/java/com/github/jferard/fastods/TableAppender.java

```diff
@@ -128,13 +127,6 @@ private void appendShapes(final XMLUtil util, final Appendable appendable,
         appendable.append("</table:shapes>");
     }
 
-    private void appendForms(final XMLUtil util, final Appendable appendable) throws IOException {
```

Comment:
```
appendable.append("<office:forms");
...
appendable.append(">");
for (XMLConvertible form : this.forms) {
...
}
appendable.append("</office:forms>");

```
This has no cost unless forms are implemented and used.

Reply:
ok :)